### ISSUE TITLE
Request for new Gedcom Tag

### ISSUE THREAD

**Issue by GCLynda** [ShammyLevva/FTAnalyzer#121]
Could you consider adding the GEDCOM Tag please

Tag    | formal description | english description
EVEN | EVENT | A noteworthy happening related to an individual, a group, or an organization.

I personally use it a lot and imagine others do to so it would be fantastic if it were added as an acceptable code rather than displaying on the Exceptions report.

Love the programme :-)

Thank you...
Lynda

**Comment by ShammyLevva** [comment]
Sorry I don’t understand what you mean. The EVENt tag is what is in GEDCOM as a custom tag. When you add your own custom tag it can be anything. 

So to in order to understand what you mean I’d need to see a sample GEDCOM. However I suspect you mean a custom tag

**Comment by GCLynda** [comment]
Hi Alexander

apologies for the delay - I've been away working .

Yes -- I mean a tag.  I get the message under -- which I don't mind and
know can be turned off

The following unknown fact types were reported.
NB. This isn't an error if you deliberately created these fact types.
This is simply highlighting the types so you can check for any possible
errors/duplicate types.


[image: image.png]

But I don't understand why EVENT is flagged as an unknown fact in FTA.

[image: image.png]

An example from my GED file is

2 NOTE @N34@
1 EVEN Visiting family members and meeting distant cousins for the fist time
2 TYPE Event
2 DATE BET 16 AUG AND 15 SEP 2013
2 PLAC United Kingdom
3 MAP
4 LATI N52.3394
4 LONG W2.2108

and another

1 EVEN Aunt Shirley Beard's wedding to Tony Rullo
2 TYPE Event
2 DATE 3 JUL 1954
1 EVEN step-Lisa Sincock's wedding to Anthony Gibbs
2 TYPE Event

Hopefully, this is something that can be changed so that it is recognised?

Cheers
Lynda :-)

**Comment by ShammyLevva** [comment]
Yes. It seems you are not understanding what the use of the EVENT tag is. It’s entire purpose is to add custom tags that aren’t in the original GEDCOM standard. So by definition an EVENt tag is a custom tag that is user definable. 

Thus as I say I have no idea what you are wanting. The point of the event tag is to create new fact types however in your usage you aren’t using them for a custom type.

A custom type might be some specific event that might occur that you want to tag lots of people with. A custom event type is not a note. Yet you are using it like a note against someone. 

The example 
1 EVEN Aunt Shirley Beard's wedding to Tony Rullo 
2 TYPE Event 
2 DATE 3 JUL 1954

Could not be a better example of precisely what you absolutely should NEVER do. This is a MARRiage fact related to two people who were married, not an event. Even if for some unfathomable reason you wanted a fact type that you could say who attended a wedding. Then you would create a fact type for attended wedding. 

Then when you use that custom fact type you would note the date and the name of the wedding and tag each person you wanted to to add the event to. What you absolutely DO NOT do is create fact type after fact type for different weddings. The fact type would be ATTENDED WEDDING regardless of which wedding it was. 

So basically the Event tag is doing EXACTLY what it should be doing. It is showing you where you have used custom tags so you can decide if you have misused them or not. If you have as per your example multiple tags for the same thing you’d see that and be able to fix it.

**Comment by GCLynda** [comment]
I'm so new to all this and appreciate your guidance hugely.

I've been using EVENt to log family members who were present in some way at
something - ie a sibling wedding, informant at the death of a family member
- that kind of thing.

I'll revisit in light of your clarification of usage.

Many thanks!

Cheers
Lynda

**Comment by ShammyLevva** [comment]
Yup so a custom fact for witnessing an event would be a WITNESS fact. This is so common that it’s a recognised fact type. Yet it’s not in the GEDCOM standard so would be an event. 

The reason that you may be confused is that who attended a wedding doesn’t tell you much about a birth, marriage or death and those along with a census are your core fact types. 

If you think about it you only need to go back 50 odd years to completely struggle to find any reliable info on who attended a wedding other than the bride and groom and perhaps a witness. Although in many documents the only witnesses recorded will be the bride and groom and the minister. 

So it would only be the most recent of weddings that you’d find any info on who attended.

**Comment by GCLynda** [comment]
this is true.... sigh... lol

I'll review where I've used EVENt and ascertain if it's a better option to
amend to WITNESS - it sounds as it it would be for sure...

Cheers
